samples/gitchurn/src/gitChurnMain/kotlin/GitChurn.kt

```diff
@@ -35,11 +35,14 @@ fun main(args: Array<String>) {
 private fun calculateChurn(workDir: String, limit: Int) {
     println("Opening…")
     val repository = git.repository(workDir)
-    val map = mutableMapOf<String, Int>()
+    //val map = mutableMapOf<String, Int>()
+    val namedMap = mutableMapOf<Pair<String, String>, Int>()
     var count = 0
-    val commits = repository.commits()
-    val limited = commits.take(limit)
-    println("Calculating…")
+    //val commits = repository.commits()
+    val myCommits = repository.myCommits()
+    //val limited = commits.take(limit)
+    val myLimited = myCommits.take(limit)
```

Comment:
Names like `mySomething` should be avoided. "My" doesn't help to understand the purpose of the variable here.